package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "zls-wasm",
+  "name": "playground",
```

Comment:
Thanks, I updated it there too now. I think the name in `package-lock.json` got updated to the containing directory since no name was specified in `package.json`; this should now be more consistent.